xbmc/network/Network.h

```diff
@@ -145,10 +181,122 @@ class CNetwork
    void StartServices();
    void StopServices(bool bWait);
 
+   /*!
+    \brief  Tests if parameter specifies valid IPv6 address (as specified by RFCs)
+    \param  IP as std::string, struct sockaddr* (non mandatory),
```

Comment:
name the param so the format is
\param address description
\param sa description
\param port description